### PR TITLE
Small tweaks

### PR DESCRIPTION
**Pull request by billyroh**
* Add Roboto Mono as a backup
* Make the width a bit wider
* Differentiate time label from event label
* Add favicon

cc @visnup

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,7 +5,9 @@ const head = document.createElement('div');
 head.innerHTML = `
   <link href='http://fonts.googleapis.com/css?family=PT+Sans' rel='stylesheet' type='text/css'>
   <meta name="viewport" content="initial-scale=1" />
-  <title>waffle.js</title>
+  <title>WaffleJS</title>
+  <link rel="icon" type="image/png" href="favicon.png" sizes="16x16">
+  <link rel="icon" type="image/png" href="favicon@2x.png" sizes="32x32">
```
Comment (visnup) [inline index.js]: probably only need the 2x version and let the browser scale it down.

File: index.html [hunk 2]
```diff
@@ -17,12 +17,12 @@
...
-* 7:25 PM -- Second speaker · [Submit Talk][4]
-* 7:40 PM -- Third speaker · [Submit Talk][4]
-* 7:55 PM -- Fourth speaker · [Submit Talk][4]
-* 8:10 PM -- Hang out
+* <span class="time">7:00 PM</span> -- Introductions
+* <span class="time">7:10 PM</span> -- First speaker · [Submit Talk][4]
+* <span class="time">7:25 PM</span> -- Second speaker · [Submit Talk][4]
+* <span class="time">7:40 PM</span> -- Third speaker · [Submit Talk][4]
+* <span class="time">7:55 PM</span> -- Fourth speaker · [Submit Talk][4]
+* <span class="time">8:10 PM</span> -- Hang out
```
Comment (visnup) [inline index.html]: I'd do this with bold or italics (`**` or `*` or `__` or `_`) and style using that.

Comment (visnup) [inline index.html]: markdown hacks!!

Comment (billyroh) [inline index.html]: :weary:

**Comment by billyroh** [comment]
@visnup ok i made the changes

**Pull request merged**